test/Microsoft.AspNet.Mvc.FunctionalTests/FiltersTest.cs

```diff
@@ -155,7 +155,7 @@ public class FiltersTest
                 "http://localhost/AuthorizeUser/Impossible");
 
             // Assert
-            Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
+            Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
```

Comment:
This is actually finally the correct behavior, when a policy fails with a missing claim, you want 403 forbidden (not unauthorized).  This finally fixes it (for one path).  There's still an issue which currently isn't shown in the mvc tests)